css-flexbox/css-flexbox_profile/css/styles.css

```diff
@@ -31,6 +31,8 @@ body {
 }
 
 .profile__image {
+  display: block;
```

Comment:
as we saw in the session, we can use `align-self: center` here :)

Reply:
Yes, you're right